File: python/baiduads-sdk/baiduads_manual/oauth/model/get_open_id_request.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
"""
Copyright (c) 2022 Baidu.com, Inc. All Rights Reserved
"""


class GetOpenIdRequest:
    """get open id request data model

    Attributes:
        accessToken: 授权令牌accessToken
        userId: 同意授权用户ucid
    """

    def __init__(self, accessToken, userId):
        self.accessToken = accessToken
        self.userId = userId

File: java/baiduads-sdk/src/main/java/com/baidu/dev2/api/sdk/manual/oauth/model/GetOpenIdRequest.java
package com.baidu.dev2.api.sdk.manual.oauth.model;


public class GetOpenIdRequest {
    /**
     * 授权令牌accessToken
     */
    private String accessToken;

    /**
     * 同意授权用户ucid
     */
    private Long userId;

    public String getAccessToken() {
        return accessToken;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }
}
